src/main/kotlin/ua/marchenko/artauction/common/annotation/scheduled/annotation/CustomScheduled.kt

```diff
@@ -0,0 +1,5 @@
+package ua.marchenko.artauction.common.annotation.scheduled.annotation
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+annotation class CustomScheduled(val schedule: String)
```

Comment:
how does another programmer know what to pass as a parameter for your annotation? 

cron? ISO timestamp? fixed delays in milliseconds? At the current moment, I do not fully understand the idea of DayTimeDetails but if it is expected in an annotation processor then you may make it a parameter type